### ISSUE TITLE
Maintain <genreForm> as a separate element

### ISSUE THREAD

**Issue by BrigitteMichel** [SAA-SDT/eas-schemas#145]
## Creator of issue

1. Brigitte Michel (michel@abes.fr)
2. Abes (Agence bibliographique de l'enseignement supérieur = Bibliographic Agency of Higher Education)

## The issue relates to

- [ ] EAC-CPF schema issue
- [ ] EAC-CPF Tag Library issue
- [x] EAD schema issue
- [x] EAD Tag Library issue
- [ ] Schema issue
- [ ] Tag Library issue
- [ ] Suggestions for all schemas
- [ ] Suggestions for all Tag Libraries
- [ ] Other

## Wanted change/feature

* Text: Removing `<genreform>` seems very problematic to us.

1. `<genreform>`, as practiced in EAD 2002 by French university libraries, is highly codified ([source](https://documentation.abes.fr/aidecalames/manuelcatalogageead/index.html#IndexationGenresCaracteristiquePhysiques:K1mwsYcpQJGEHWyE7gAyg)). There are 3 controlled values for `@type`, and for each TYPE value, there is a controlled list of NORMAL values: [List 1: GenreformDocumentType](https://documentation.abes.fr/aidecalames/manuelcatalogageead/index.html#GenreformType); [List 2: GenreformTechnique](https://documentation.abes.fr/aidecalames/manuelcatalogageead/index.html#GenreformTypeTechnique_); [List 3: GenreformGFF](https://documentation.abes.fr/aidecalames/manuelcatalogageead/index.html#GenreformTypeGenre).
2. The concept of `<genreform>` is gaining international traction. The latest list mentioned above is soon to be replaced by a relationship to a new national reference framework defined within the LRM/RDA program as a "form of the work" (category of Work) (source RDA: [Glossary](https://access.rdatoolkit.org/Glossary?externalId=en-US_rdaregistry.info-Elements-w-P10004_see_1), [RDA Toolkit](https://access.rdatoolkit.org/Content/Index?externalId=en-US_ala-28e39ed9-43ec-3a0d-bd30-83c805efaa8c&highlight=form&highlight=of&highlight=work). It also exists in Bibframe ([source](https://id.loc.gov/ontologies/bibframe.html#p_genreForm)).
3. The solution proposed by EAD 4.0 (`<subjectHeadings>`) goes against the international movement aiming to distinguish the form of the document and/or its content (letters, reports, photos, interviews) from the subject matter in the document.
4. There are several concepts in RiC that justify a distinct element: RiC: RiC-A10 Content Type and RiC-A17 Documentary Form Type, but it could also possibly concern RiC-A39 State. 

## Suggested Solution

* Text: Define `<genreForm>`  (possibly `<genresForms>`) to occur within `<archDes>`, `<c>`, `<c01>` etc. or better in  `<identificationData>`
*  Attributes: audience, conventionDeclarationReference, id, languageOfElement, localType, localTypeDeclarationReference, maintenanceEventReference, scriptOfElement, sourceReference, target, valueURI, vocabularySource, vocabularySourceURI
* Plus `@ricType` and `@ricId` (see suggestion in #150)
* Plus `@coverage` (see #146 for this specific aspect)
* Occurrence: optional, repeatable

## Context

* Text:  NB List 2 could become a new `<physFacet>` but Lists 1 and 3 are about the record resource (RiC-E01), not an instantiation (RiC-E06).

**Comment by florenceclavaud** [comment]
>     3. The solution proposed by EAD4 (`<subjectHeadings>`) goes against the international movement aiming to distinguish the form of the document and/or its content (letters, reports, photos, interviews) from the subject matter in the document.

I agree with this remark. The description and usage note of `<subjectHeadings>` in EAD 4 says: "Use <subjectHeadings> to bundle topical or thematic access points that represent the contexts of the materials described." 
While the state of a record (RiC-A39 State; e.g. copy, original, draft, etc.), its form (RiC-A17 Documentary form type; e.g. bull, charter, birth certificate...; and also what diplomatics calls 'nature juridique', in English 'legal nature', e.g. gift, grant, endourment...) are definitely not topics. They categorize the records described.

The name and application note of this element, that has replaced EAD 2002 `<controlaccess>`, reduce its scope.
IMHO this element should be renamed and/or (at least) the Tag Libray modified; or `<genreform>` or a replacing element should be reintroduced directly in `<c>` or `<identificationData>`.

> 
>     4. There are several concepts in RiC that justify a distinct element: RiC: RiC-A10 Content Type and RiC-A17 Documentary Form Type, but it could also possibly concern RiC-A39 State.

Yes, I agree too; see my reaction above.
I would add that <genreform> was also used to categorize groups of records, not only items. So RiC-A36 Record Set Type can be added to this list of attributes.
>

**Comment by MicheleCombs** [comment]
I agree that retaining <genreform> makes sense, for all the reasons given above.  It seems to me that it would be both practically damaging and semantically suspect to force the intellectual and the physical into a single element.

**Comment by kerstarno** [comment]
This will be discussed by TS-EAS during their meeting on 12/13 August 2024.
